packages/components/nodes/embeddings/GoogleVertexAIEmbedding/GoogleVertexAIEmbedding.ts

```diff
@@ -58,7 +82,9 @@ class GoogleVertexAIEmbedding_Embeddings implements INode {
 
             if (projectID) authOptions.projectId = projectID
         }
-        const obj: GoogleVertexAIEmbeddingsParams = {}
+        const obj: GoogleVertexAIEmbeddingsParams = {
+            model: modelName
```

Comment:
if `modelName` is undefined, is it still running fine? maybe a better way is to check:
```
if (modelName) obj.model = modelName
```

Reply:
Oh yeah!

you are correct!
thank you for pointing out!